packages/ember-intl/addon-test-support/add-translations.ts

```diff
@@ -15,14 +14,7 @@ export async function addTranslations(
   locale: string,
   translations: Translations,
 ): Promise<void> {
-  const context = getContext();
-
-  assert(
-    'To use `addTranslations()`, make sure to call `setupTest()`, `setupRenderingTest()`, or `setupApplicationTest()`.',
-    context,
-  );
```

Comment:
I decided to remove `assert()`, as it's unlikely that developers can run a test without `setupTest()`, `setupRenderingTest()`, or `setupApplicationTest()`.

Note, the `assert()` had provided a more user-friendly error message.